locales/fr.lua

```diff
@@ -0,0 +1,24 @@
+local Translations = {
+    error = {
+        not_authorized = "Pas autorisé",
+        lockpick_fail = "Échoué",
+        screwdriverset_not_found = "Vous n'avez pas de kit de tournevis",
+        door_not_locked = "La porte n'est pas vérouillée",
+        door_not_lockpickable = "Cette porte ne peut pas être crochetée"
+    },
+    success = {
+        lockpick_success = "succès"    },
+    general = {
+        locking = "~r~verrouillage..",
```

Comment:
S'il vous plait, can you make make the first letter capitalized? Succes, Verrouillage, etc...

Merci!!